<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CViewAbout</class>
 <widget class="QWidget" name="CViewAbout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>关于</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="6" column="0">
      <widget class="QLabel" name="labelBuild">
       <property name="text">
        <string>HMI程序生成日期：</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="labelDbVer">
       <property name="text">
        <string>数据库版本：</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelCRC">
       <property name="text">
        <string>HMI程序校验码：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>SSP01系统</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelVer">
       <property name="text">
        <string>HMI程序版本：</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="labelDbInst">
       <property name="text">
        <string>历史主数据库实例：</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="labelDbInst_2">
       <property name="text">
        <string>历史备数据库实例：</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="labelMDbInst">
       <property name="text">
        <string>内存数据库实例：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>江苏金智科技股份有限公司</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>南京悠阔电气科技有限公司</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>201</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ssp_gui.qrc"/>
 </resources>
 <connections/>
</ui>
